docs/transient-crr-source.md

```diff
@@ -0,0 +1,177 @@
+# Transient CRR Source
+
+## Description
+
+The goal of this enhancement is to save the cost of reading data from
```

Comment:
I think we can use the word "avoid" instead of "save" here so it's not interpreted as if there is still some cloud charge involved in the read.